docker/docker_image_src.go

```diff
@@ -39,6 +39,20 @@ func newImageSource(ctx *types.SystemContext, ref dockerReference, requestedMani
 	if requestedManifestMIMETypes == nil {
 		requestedManifestMIMETypes = manifest.DefaultRequestedManifestMIMETypes
 	}
+	var (
+		acceptableRequestedMIMEs = true
+		supportedMIMEs           = supportedManifestMIMETypesMap()
+	)
+	for _, mtrequested := range requestedManifestMIMETypes {
+		if supportedMIMEs[mtrequested] {
+			continue
+		}
+		acceptableRequestedMIMEs = false
+		break
```

Comment:
(i.e. when reaming `reset` to `acceptableRequestedMIMEs`, only the last `if` was flipped AFAICS.)